js/webapp.js

```diff
@@ -935,7 +935,6 @@ WebApp.prototype = {
       this._installPluginStaticHandlers(plugin, urlBase);      
       //import resolution will be postponed until all non-import plugins are loaded
       //only push plugin if no exceptions were seen
```

Comment:
I think line 937 is no longer relevant after your deletion. You can decide about line 936...

Reply:
I'll move it.. it's still relevant but in the wrong place by a few lines.